cves/kernel/CVE-2021-29647.yml

```diff
@@ -450,7 +443,7 @@ mistakes:
 
     Write a thoughtful entry here that people in the software engineering
     industry would find interesting.
-  answer:
+  answer: This was most likely a lapse as initializing is a simple thing that was most likely overlooked
```

Comment:
This section leaves a lot to be desired. You should try to create a narrative out of your vulnerability. Why did it occur? How could it have been prevented? What lessons can be learned? Why does anyone care? Try to make it interesting to someone who is reading about the vulnerability, but isn't a security expert.